source/Ansible-Protocol-091/Amqp091ConfirmSelect.class.st

```diff
@@ -0,0 +1,57 @@
+Class {
```

Comment:
Also a protocol extension for RabbitMQ

```
Using standard AMQP 0-9-1, the only way to guarantee that a message isn't lost is by using transactions -- 
make the channel transactional then for each message or set of messages publish, commit. In this case, 
transactions are unnecessarily heavyweight and decrease throughput by a factor of 250. To remedy this, a
confirmation mechanism was introduced. It mimics the consumer acknowledgements mechanism already
present in the protocol.
```

See https://www.rabbitmq.com/confirms.html#publisher-confirms